src/reporter/index.js

```diff
@@ -31,7 +31,8 @@ export default class Reporter {
             errs:           [],
             unstable:       false,
             startTime:      null,
-            testRunInfo:    null
+            testRunInfo:    null,
+            contexts:       []
```

Comment:
There are two different contexts: fixture and test. 
See [help topic](https://devexpress.github.io/testcafe/documentation/test-api/test-code-structure.html#fixture-hooks) for more detail.
So instead of a single `context` prop, please add two properties: `ctx` and `fixtureCtx`.